lib/JSON/Validator.pm

```diff
@@ -1294,7 +1294,7 @@ Used to create a new schema, where the C<$ref> are resolved. C<%args> can have:
 
 =over 2
 
-=item * C<{replace => 1}>
+=item * C<< { replace => 1 } >>
```

Comment:
I don’t fancy the extra white space. Please remove it so it looks like this: `<<{replace => 1}>>`.

Reply:
done.